imap_processing/hit/l0/hit_l1a_decom.py

```diff
@@ -0,0 +1,110 @@
+import logging
+from collections import defaultdict
+from enum import IntEnum
+
+import xarray as xr
+
+from imap_processing import decom
+
+logging.basicConfig(level=logging.INFO)
+
+
+class HitAPID(IntEnum):
+    HIT_AUT = 1250  # Autonomy
+    HIT_HSKP = 1251  # Housekeeping
+    HIT_SCIENCE = 1252  # Science
+    HIT_IALRT = 1253  # I-ALiRT
+    HIT_MEMDUMP = 1255  # Memory dump
+
+
+def decom_hit_packets(packet_file: str, xtce: str):
+    """
+    Unpack and decode HIT packets using CCSDS format and XTCE packet definitions.
+    Parameters
+    ----------
+    packet_file : str
+        Path to the CCSDS data packet file.
+    xtce : str
+        Path to the XTCE packet definition file.
+    Returns
+    -------
+    dict
+        A dictionary containing xr.Dataset for each APID. each dataset in the
+        dictionary will be converted to a CDF.
+    """
+    # TODO: XTCE Files need to be combined
+    logging.info(f"Unpacking {packet_file} using xtce definitions in {xtce}")
+    packets = decom.decom_packets(packet_file, xtce)
+    logging.info(f"{packet_file} unpacked")
+
+    # sort all the packets in the list by their spacecraft time
+    sorted_packets = sorted(packets, key=lambda x: x.data["SHCOARSE"].derived_value)
+
+    unpacked_data = defaultdict(list)
```

Comment:
```suggestion
    # Store data for each apid
    # unpacked_data = {apid0: {var0: [item0, item1, ...], var1: [item0, item1, ...]}, ...}
    unpacked_data = {}
```
I think this one should just be a dictionary? So unpacked data will be a dict of dicts now I believe.